README.md

```diff
@@ -513,67 +513,116 @@ from events
 group by 1
 ```
 
-## Iceberg
+#### Iceberg
 
-The adaptor support Iceberg as an Open Table Format. In order to work with Iceberg,
-setup an Iceberg connector and then create an Iceberg connection in Glue connections.
-Then use the following `conf` in your profile
+**Usage notes:** The `merge` with Iceberg incremental strategy requires:
+- To attach the AmazonEC2ContainerRegistryReadOnly Manged policy to your execution role :
+- To add the following policy to your execution role to enable commit locking in a dynamodb table (more info [here](https://iceberg.apache.org/docs/latest/aws/#dynamodb-lock-manager))
 ```
-connections: iceberg_connection
-conf: "spark.sql.catalog.iceberg_catalog=org.apache.iceberg.spark.SparkCatalog --conf spark.sql.catalog.iceberg_catalog.warehouse=s3://your_bucket/prefix --conf spark.sql.catalog.iceberg_catalog.catalog-impl=org.apache.iceberg.aws.glue.GlueCatalog --conf spark.sql.catalog.iceberg_catalog.io-impl=org.apache.iceberg.aws.s3.S3FileIO --conf spark.sql.extensions=org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions --conf spark.sql.sources.partitionOverwriteMode=dynamic --conf spark.sql.iceberg.handle-timestamp-without-timezone=true"
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "CommitLockingTableFullAccess",
+            "Effect": "Allow",
+            "Action": "dynamodb:*",
+            "Resource": "arn:aws:dynamodb:<AWS_REGION>:<AWS_ACCOUNT_ID>:table/myGlueLockTable"
```

Comment:
Con you add a comment before the policy saying that  myGlueLockTable should be the same than on the config file?

Reply:
I think that you are missing that the user needs to create a DynamoDB table. You could recommend to use On-Demand pricing. Maybe create a ticket (enhancement) to add a cloudformation with an example

Reply:
In fact, the DynamoDB table is automatically created with On-Demand pricing by default (see [here] (https://iceberg.apache.org/docs/latest/aws/#dynamodb-lock-manager)). But, you're right the permission should be on the same table that is specified in the config file. I'll add a comment on that in the next PR, as well as instructions and code refactoring for users that want to use their own created DynamoDB Table, or to use another name for this table.